test/Tests/src/Data/Numbers_Spec.enso

```diff
@@ -518,6 +518,19 @@ spec =
             3.1416 . round 3 . should_equal 3.142
             3.9999 . round 3 . should_equal 4.0
 
+        Test.specify "Can round negative decimals to a specified number of decimal places" <|
```

Comment:
To ensure the behavior of `RoundNode` builtin and the `Column.round` is the same, can we generalize this `Numbers_Spec` and run it twice on the same values? Once using regular `round` builtin, once creating a `Column` with the same value and rounding via `Column.round`? I assume the result should be the same.

Reply:
Done.